README.md

```diff
@@ -33,9 +46,13 @@ make deps
 make run
 ```
 
-## Local (Development)
+## Docker installation and usage
+
+### Requirements
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)